lldb/include/lldb/Core/Progress.h

```diff
@@ -69,7 +69,8 @@ class Progress {
   ///
   /// @param [in] debugger An optional debugger pointer to specify that this
   /// progress is to be reported only to specific debuggers.
-  Progress(std::string title, uint64_t total = UINT64_MAX,
+  Progress(std::string title, std::string details = {},
+           uint64_t total = UINT64_MAX,
```

Comment:
Might be good to start using `std::optional<uint64_t>` internally in the `Progress` class and avoid the magic numbers as long as we are changing things up

Reply:
For sure, this was actually a part of the original patch I made trying to introduce the aggregate flag.